cumulusci/tasks/salesforce/trigger_handlers.py

```diff
@@ -0,0 +1,91 @@
+import yaml
+import os
+from collections import defaultdict
+
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg, process_bool_arg
+from cumulusci.core.exceptions import TaskOptionsError, CumulusCIException
+
+
+class SetTDTMHandlerStatus(BaseSalesforceApiTask):
+    """Alter the active status of specific NPSP or EDA TDTM trigger handlers.
+    Save a state file to allow the handlers to be restored to their previous status.
+    """
+
+    task_options = {
+        "handlers": {
+            "description": "List of Trigger Handlers (by Class, Object, or 'Class:Object') to affect (defaults to all handlers)."
+        },
+        "namespace": {
+            "description": "The namespace of the Trigger Handler object ('eda' or 'npsp'). The task will apply the namespace if needed."
+        },
+        "active": {
+            "description": "True or False to activate or deactivate trigger handlers."
+        },
+        "restore_file": {
+            "description": "Path to the state file to store the current trigger handler state."
+        },
+        "restore": {
+            "description": "If True, restore the state of Trigger Handlers to that stored in the restore file."
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super(SetTDTMHandlerStatus, self)._init_options(kwargs)
+        self.options["handlers"] = process_list_arg(self.options.get("handlers", []))
+        self.options["active"] = process_bool_arg(self.options.get("active", False))
+        self.options["restore"] = process_bool_arg(self.options.get("restore", False))
+
+        if self.options["restore"] and not self.options.get("restore_file"):
+            raise TaskOptionsError("Restoring requires a restore file name")
+
+    def _run_task(self):
+        global_describe = self.sf.describe()
+        sobject_names = [x["name"] for x in global_describe["sobjects"]]
+
+        if self.options["restore"]:
+            with open(self.options["restore_file"], "r") as f:
+                target_status = yaml.safe_load(f)
+                self.options["handlers"] = list(target_status.keys())
+        else:
+            target_status = defaultdict(lambda: self.options["active"])
+
+        namespace = self.options.get("namespace", "npsp") + "__"
```

Comment:
I'm not totally sold on using npsp as a default here, but I don't feel strongly about it.

Reply:
No, you're right - there shouldn't be a default. I'll remove it, with the result that you can use the task with no namespace in a non-namespaced org, and it'll throw an exception asking you to specify if the org's namespaced and it cannot locate the handler object.